x/upgrade/abci_test.go

```diff
@@ -393,3 +397,26 @@ func TestUpgradeWithoutSkip(t *testing.T) {
 	VerifyDoUpgrade(t)
 	VerifyDone(t, s.ctx, "test")
 }
+
+func TestDumpUpgradeInfoToFile(t *testing.T) {
+	s := setupTest(10, map[int64]bool{})
+
+	planHeight := s.ctx.BlockHeight() + 1
+	name := "test"
+	t.Log("verify if upgrade height is dumped to file")
+	err := s.keeper.DumpUpgradeInfoToDisk(planHeight, name)
+	require.Nil(t, err)
+
+	upgradeInfoFilePath, err := s.keeper.GetUpgradeInfoPath()
+	require.Nil(t, err)
+
+	data, err := ioutil.ReadFile(upgradeInfoFilePath)
```

Comment:
Is this file removed after the test is complete? If not, can you ensure it's deleted?